frontend/src/pages/LandingPage.tsx

```diff
@@ -28,17 +28,17 @@ const LandingPage = () => {
                     </h1>
                 </div>
                 <div className="searching">
-                    <div className="inputs">
-                        <input
-                            ref={searchBar}
-                            id="rsn"
-                            name="rsn"
-                            placeholder="Enter your RuneScape Username"
-                        />
-                        <button type="button" onClick={loadAccountPage}>
-                            Search
-                        </button>
-                    </div>
+                    <form onSubmit={loadAccountPage}>
```

Comment:
why a form again?

Reply:
so after user puts something in the search bag you can press enter to load the account page